frontend/src/Editor/Components/Statistics.jsx

```diff
@@ -64,45 +72,55 @@ export const Statistics = function Statistics({ height, properties, styles, dark
 
   return (
     <div style={baseStyle}>
-      <p
-        style={{
-          ...letterStyle,
-          ...marginStyle,
-          color: primaryLabelColour !== '#8092AB' ? primaryLabelColour : darkMode && '#FFFFFC',
-        }}
-      >
-        {primaryValueLabel}
-      </p>
-      <h2 style={primaryStyle}>{primaryValue}</h2>
-      {hideSecondary ? (
-        ''
+      {loadingState === true ? (
```

Comment:
Can we just destructure `loadingState` from properties here, and not use hooks?

Reply:
Okay, working on it.

Reply:
@arpitnath made the required changes